### PR TITLE
Adds a new set of admin-only powercells intended for debugging

### PR DESCRIPTION
**Pull request by genessee596**
## About The Pull Request

Adds a set of cells with infinite charge. They have custom sprites, and never run out of charge. This makes them ideal for debugging weapons or power stuff.

Sprites by me.

image
_Above: a Bluespace Technician hard at work. On the shelf sits his new set of Nullcells._

image
_Above: Maintainer approval to make this PR_

**As stated previously, these should not be obtainable by players. They are only accessible via the admin Game Panel.**

## Why It's Good For The Game

helps coders test out energy guns and stuff without having to waste time swapping cells

## Changelog
:cl:
add: Admin cells intended for debugging and admemery
/:cl:

### CODE REVIEW

**Review by Evankhell561 — CHANGES_REQUESTED** [review]
I think you repeated the same code 3 times.

**Review by Evankhell561 — APPROVED** [review]

File: code/game/objects/items/weapons/power_cells.dm [hunk 1]
```diff
@@ -313,3 +313,33 @@
...
+	maxcharge = 9999
+	matter = list(MATERIAL_PLASMA = 8, MATERIAL_STEEL = 2, MATERIAL_PLASTIC = 2, MATERIAL_SILVER = 2, MATERIAL_URANIUM = 8)
+	
+/obj/item/weapon/cell/small/moebius/nuclear/infinite
+	name = "BSL \"Nullcell 999L\""
+	desc = "Bluespace League branded rechargeable S-standardized power cell. This strange piece of technology has the tag 'Made in Finland' on the back. It never seems to run out of charge."
+	icon_state = "infinite_s"
+	autorecharge_rate = 1
+	bad_type = /obj/item/weapon/cell/small/moebius/nuclear/infinite
+	maxcharge = 999
```
Comment (Evankhell561) [inline code/game/objects/items/weapons/power_cells.dm:344]: being that this is a debug item maybe all these power cells should have maxchage = 99999

Comment (genessee596) [reply code/game/objects/items/weapons/power_cells.dm:344]: I don't think the number really matters due to 'autorecharge_rate' being 100%. I just had them be seperate numbers to be easier to tell which is which if looking at them in a stack.

**Comment by MadmanMartian** [comment]
£5 says an admin spawns these within the first round to give to players

**Comment by genessee596** [comment]
> £5 says an admin spawns these within the first round to give to players

agreed lol

speedmerge pls

**Review by TheShown911 — COMMENTED** [review]
Don't think Finland is a thing mate

**Comment by genessee596** [comment]
> Don't think Finland is a thing mate

image
fixed

**Pull request merged**